gradio/external.py

```diff
@@ -457,8 +472,10 @@ def streamline_config(config: dict) -> dict:
         return config
 
     config = streamline_config(config)
-    api_url = "https://hf.space/embed/{}/api/predict/".format(model_name)
+    api_url = "{}/api/predict/".format(iframe_url)
     headers = {"Content-Type": "application/json"}
+    if api_key is not None:
+        headers["Authorization"] = f"Bearer {api_key}"
     ws_url = "wss://spaces.huggingface.tech/{}/queue/join".format(model_name)
```

Comment:
This should be `iframe_url/queue/join` right?

Reply:
Yes I thought I fixed this, thank you!